banderwagon/precomp_multiexp.go

```diff
@@ -98,15 +93,14 @@ func DeserializePrecomputedLagrange(reader io.Reader) (*PrecomputeLagrange, erro
 		// Deserialize the matrix
 		pcl.inner[i].matrix = make([]bandersnatch.PointAffine, rowLen)
 		for j := int64(0); j < rowLen; j++ {
-
 			pcl.inner[i].matrix[j] = bandersnatch.ReadUncompressedPoint(reader)
 		}
 	}
 
 	return &pcl, nil
 }
 
-func (p *PrecomputeLagrange) Commit(evaluations []fr.Element) *Element {
+func (p *PrecomputeLagrange) Commit(evaluations []fr.Element) Element {
```

Comment:
Not returning a pointer to avoid an allocation on a stack based variable being returned.